README.md

```diff
@@ -92,21 +103,28 @@ The body, headers, etc. for this response will be taken from the API description
 Requests to operations which expect a request body will be validated, for example: a POST with JSON.
 
 ```bash
-http --json POST http://127.0.0.1:4010/pets name=Stowford
+curl -s -D "/dev/stderr" -H "content-type: application/json" -d '{"name":"Stowford"}' http://127.0.0.1:4010/pets | json_pp
 ```
 
-This will generate an error:
+This will generate an error, conforming the [application/problem+json][rfc7807] specification:
 
 ```
-Here is the original validation result instead: [{"path":["body"],"name":"required","summary":"should have required property 'photoUrls'","message":"should have required property 'photoUrls'","severity":"error"}]
+HTTP/1.1 422 Unprocessable Entity
+content-type: application/problem+json
+content-length: 279
+Date: Wed, 08 May 2019 19:22:27 GMT
+Connection: keep-alive
+
+{
+   "title" : "Invalid request body payload",
+   "detail" : "Your request body is not valid: [{\"path\":[\"body\"],\"code\":\"required\",\"message\":\"should have required property 'photoUrls'\",\"severity\":0}]",
+   "status" : 422,
+   "name" : "https://stoplight.io/prism/errors#UNPROCESSABLE_ENTITY"
```

Comment:
`name`is invalid it should be `type`. Is the example wrong or is the code wrong?

Reply:
```{
             "type": "https://example.com/probs/out-of-credit",
             "title": "You do not have enough credit.",
             "detail": "Your current balance is 30, but that costs 50.",
             "instance": "/account/12345/msgs/abc",
             "balance": 30,
             "accounts": ["/account/12345",
                          "/account/67890"]
            }```

Reply:
The code is wrong, I called the property in the wrong way. Fixed as part of #274, 9f2b35a